commands/incase.py

```
@@ -0,0 +1,22 @@
+import discord
+import lib.sussyutils
+import lib.locareader
```

Ruff (0.11.9)

`lib.locareader` imported but unused

Remove unused import: `lib.locareader`

(F401)